Models/NHES/Systems/PrimaryHeatSystem/HTGR/HTGR_Rankine_Upload/BaseClasses.mo

```diff
@@ -0,0 +1,375 @@
+within NHES.Systems.PrimaryHeatSystem.HTGR.HTGR_Rankine_Upload;
+package BaseClasses
```

Comment:
HTGR_Rankine_Upload doesn't exist when the Modelica folders load up. Please delete from the files on your branch and repush to eliminate this call.